### PR TITLE
Fix fake_handler test in Go 1.6

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: pkg/util/testing/fake_handler_test.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 )
 
 func TestFakeHandlerPath(t *testing.T) {
-	handler := FakeHandler{}
+	handler := FakeHandler{StatusCode: 200}
```
Comment (eparis) [inline pkg/util/testing/fake_handler_test.go]: Not that it really matters, but should this be http.StatusOK ?

Comment (wojtek-t) [inline pkg/util/testing/fake_handler_test.go]: Done.

**Comment by wojtek-t** [comment]
PTAL

**Comment by luxas** [comment]
@wojtek-t Thanks for working on `go1.6` compability! :)

**Pull request merged**